sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/SubscriptionProperties.cs

```diff
@@ -23,7 +23,7 @@ public class SubscriptionProperties : IEquatable<SubscriptionProperties>
         private string _userMetadata;
 
         /// <summary>
-        /// Initializes a new instance of SubscriptionDescription class with the specified name and topic name.
+        /// Initializes a new instance of SubscriptionProperties class with the specified name and topic name.
```

Comment:
```suggestion
        /// Initializes a new instance of <see cref="SubscriptionProperties" /> class with the specified name and topic name.
```

Not that it's particularly helpful, but most of the other classes are following this pattern, so in the name of consistency....